zerver/openapi/python_examples.py

```diff
@@ -728,6 +728,44 @@ def toggle_mute_topic(client: Client) -> None:
     validate_against_openapi_schema(result, "/users/me/subscriptions/muted_topics", "patch", "200")
 
 
+@openapi_test_function("/user_topics:post")
+def update_user_topic(client: Client) -> None:
+    stream_id = client.get_stream_id("Denmark")["stream_id"]
+
+    # {code_example|start}
+    # Mute the topic "dinner" in the stream having id 'stream_id'.
+    request = {
+        "stream_id": stream_id,
+        "topic": "dinner",
+        "visibility_policy": 1,
```

Comment:
This should be a follow-up, but once this is done, we should add an enum class to the `python-zulip-api` API bindings project so that this code can use a declarative name for the value imported from the module -- it'll make this example feel a lot nicer to read.